README.md

```diff
@@ -30,7 +30,7 @@ This installs the `aspect` command and also links it to `bazel`, just like the [
 
 > We plan to have a standard "core" homebrew formula so this will just be `brew install aspect` in the future.
 
-## Bazelisk (MacOS / Linux)
```

Comment:
Whoops. I keep making changes to this file instead of to silo since it has a different name in silo 😞

Reply:
To resolve this should this revert be landed and then the change made to the silo SoT?

Reply:
Interesting this diff became part of my commit? 🤔

Reply:
https://github.com/aspect-build/silo/pull/2552